src/main/resources/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep/config.jelly

```diff
@@ -40,8 +40,8 @@
             <f:enum>${it.displayName}</f:enum>
         </f:entry>
 
-        <f:entry title="Logs Folder" field="logFolder" description="The path in which the logs will be saved.">
-            <f:textbox value="${instance.logFolder}"/>
+        <f:entry title="Logs Folder" field="logFolder" description="The path in which the logs will be saved. (relative to workspace)">
+            <f:textbox value="${instance.logFolder}" default="$${WORKSPACE}/sl-log"/>
```

Comment:
Please set the default to `sealights-logs` and make sure that the filename will be `jenkins-cli.log`.

Reply:
Per the folder name - done.
Per the file name, the logs are coming from the agent which named `sl-java-agent.log`, more clear than jenkins-cli